packages/orbit-components/playroom/FrameComponent.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from "react";
+import { OrbitProvider, defaultTheme } from "@kiwicom/orbit-components";
+
+import "./index.css";
+
+const FrameComponent = ({ children }) => {
+  return (
+    <OrbitProvider theme={defaultTheme} useId={React.useId}>
+      <div id="playroom-frame" dir="ltr">
```

Comment:
It looks like `playroom-frame` is not used anywhere. Is it necessary to specify the `id` and `dir` attributes here?

Reply:
The `dir` yes. Some components rely on it. The id is just a good practice. Doesn't harm to keep. It helps with DOM debug.